### ISSUE TITLE
SMTP connection is not possible without encryption.

### ISSUE THREAD

**Issue by IgorA100** [YetiForceCompany/YetiForce#57]
This is because by default PHPMailer.php has TLS encryption enabled.
If the SMTP server does not support encryption, it is not possible to connect to it.
```
    /**
     * Whether to enable TLS encryption automatically if a server supports it,
     * even if `SMTPSecure` is not set to 'tls'.
     * Be aware that in PHP >= 5.6 this requires that the server's certificates are valid.
     *
     * @var bool
     */
    public $SMTPAutoTLS = true;
```

If encryption is not specified in the SMTP settings, then SMTPAutoTLS should be disabled
**Solution:**
In the file "\app\Mailer.php" change the code:

**BEFORE**:
```
		$this->mailer->SMTPSecure = $this->smtp['secure'];
		$this->mailer->SMTPAuth = (bool) $this->smtp['authentication'];
```
**AFTER**:
```
		$this->mailer->SMTPSecure = $this->smtp['secure'];
		if (!$this->smtp['secure']) {
			$this->mailer->SMTPAutoTLS = false;
		}
		$this->mailer->SMTPAuth = (bool) $this->smtp['authentication'];

```

**Comment by IgorA100** [comment]
@rskrzypczak 
Isn't this a bug?
YetiForce has an option to disable encryption in its settings, but this option doesn't work correctly.

**Comment by rskrzypczak** [comment]
@IgorA100 
Sorry, it's quite rare to allow without encryption. Most people don't know what to set, so they don't set anything and are happy that it works :)
Nevertheless, you are absolutely right about that. Thanks a lot.

**Comment by IgorA100** [comment]
> it's quite rare to allow without encryption. Most people don't know what to set, so they don't set anything and are happy that it works :)

Yes, I agree. Now 90% or more use encryption. And few people think about the settings....
But, for example, on their own servers, encryption may not be used in some cases.
Thanks!

**Comment by rskrzypczak** [comment]
Fixed in YF 7.0.4 ;)
Thanks a lot @IgorA100
